os/kernel/log_dump/log_dump.c

```diff
@@ -75,7 +83,11 @@ static int curbytes = 0;		/* number of bytes in the current chunk that were fill
 static int current_ptr = 0;		/* current pointer for read */
 static size_t log_dump_size = 0;
 static struct log_dump_chunk_s *read_node;
-
+static signed char *uncomp_array;
+static int uncomp_curbytes = 0;
+static char c_size[5];
+unsigned int propsSize = LZMA_PROPS_SIZE;
```

Comment:
This is too LZMA specific. Is it possible to make it as adaptive layer?

Reply:
yes, I will change it with miniz support PR